### PR TITLE
Documentation Fix for 0.2.0

### LINKED ISSUE #285: Reorder The modules list in docs(symbolic part)
🐞 **Problem**
The ordering looks too arbitrary & doesn't make sense.

🎯 **Goal**

  - Reorder them

💡 **Possible solutions**

  - Maybe in this fashion :
    - Metric
    - Christoffel
    - Riemann
    - Ricci

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### LINKED ISSUE #289: The Interactive Plots do not show up in notebook!
🐞 **Problem**

🎯 **Goal**

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by ritzvik**
Fixes #285

### CODE REVIEW

**Comment by ritzvik** [comment]
The only solution I could see regarding the plotly plot in jupyter notebook is render it as an image.
I tried to inject javascript into the notebook and it works fine, but it isn't rendered by RTD sphinx.

https://einsteinpy-rtd.readthedocs.io/en/trials/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.html

Here is the commit https://github.com/ritzvik/einsteinpy/commit/d5dfa50ded0e8a9c99860b92633110cf50900ea5

and it works fine in nbviewer https://nbviewer.jupyter.org/github/ritzvik/einsteinpy/blob/d5dfa50ded0e8a9c99860b92633110cf50900ea5/docs/source/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.ipynb

File: docs/source/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.ipynb [hunk 1]
```diff
@@ -119,6 +79,13 @@
     "obj.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![plot](./perihelion_advance.png)"
```
Comment (shreyasbapat) [inline docs/source/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.ipynb]: What?

Comment (ritzvik) [reply docs/source/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.ipynb]: > The only solution I could see regarding the plotly plot in jupyter notebook is render it as an image.
> I tried to inject javascript into the notebook and it works fine, but it isn't rendered by RTD sphinx.
> 
> https://einsteinpy-rtd.readthedocs.io/en/trials/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.html
> 
> Here is the commit [ritzvik@d5dfa50](https://github.com/ritzvik/einsteinpy/commit/d5dfa50ded0e8a9c99860b92633110cf50900ea5)
> 
> but it works fine in nbviewer https://nbviewer.jupyter.org/github/ritzvik/einsteinpy/blob/d5dfa50ded0e8a9c99860b92633110cf50900ea5/docs/source/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.ipynb

. see this

**Comment by shreyasbapat** [comment]
This is not a solution! Let's leave it as of now :/

**Comment by ritzvik** [comment]
> This is not a solution! Let's leave it as of now :/

I would say, better than nothing!

**Comment by ritzvik** [comment]
Yo @shreyasbapat ,
It is rendering now https://einsteinpy-rtd.readthedocs.io/en/bugfix2/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.html .
beacuse of your commit https://github.com/einsteinpy/einsteinpy/commit/1dbec7dfab7cf27167836fad9fccc2786189b46e ?

PR updated

**Comment by shreyasbapat** [comment]
Yes. because of my commit! Hehe

**Comment by shreyasbapat** [comment]
I removed all the errors that JS was throwing in console

**Comment by shreyasbapat** [comment]
Oh Oh! Its @varunvaruns9 Lol! 

I will make a single commit making some changes in the plotting module. How can I unsee that!

**Comment by Varunvaruns9** [comment]
I only tested it locally, so missed this. 😅

**Comment by ritzvik** [comment]
Any changes required to this PR?

**Comment by ritzvik** [comment]
Rolled back the changes in notebook example

**Pull request merged**